tendrl-monitoring-integration.spec

```diff
@@ -9,7 +9,7 @@ License: LGPLv2+
 URL: https://github.com/Tendrl/monitoring-integration
 
 Requires: tendrl-commons
-Requires: grafana
+Requires: grafana >= 4.5.2, grafana < 4.5.3
```

Comment:
Is this tested? From where the 4.5.2 package is come from?

Reply:
I have build the rpm and install the rpm which pulls 4.5.2-1 from 
https://packagecloud.io/grafana/stable/.

Reply:
Okay, then we are good to go.